modules/programs/neovim.nix

```diff
@@ -417,7 +434,8 @@ in {
       (neovimConfig // {
         wrapperArgs = (lib.escapeShellArgs neovimConfig.wrapperArgs) + " "
           + extraMakeWrapperArgs + " " + extraMakeWrapperLuaCArgs + " "
-          + extraMakeWrapperLuaArgs;
+          + extraMakeWrapperLuaArgs + " "
+          + concatStringsSep " " cfg.extraWrapperArgs;
```

Comment:
maybe put those into the escapeShellArgs 2 line 435 that has better  behavior

Reply:
I changed to add `cfg.extraWrapperArgs` to `neovimConfig.wrapperArgs`.
The example has also been modified accordingly.